examples/bin/run

```diff
@@ -29,3 +30,7 @@ if (input[0] === "transfer-ark") {
 if (input[0] === "transfer-trx") {
 	(async () => transferTrx())();
 }
+
+if (input[0] === "transfer-xlm") {
+	(async () => transferXlm())();
```

Comment:
Also possible to wraps the whole file in
```typescript
(async () => {
  //
})();
to avoid repeating the aync

Reply:
Awesome! Thanks @axeldelamarre.